### ISSUE TITLE
Add path drawing

### ISSUE THREAD

**Issue by Matt-Int** [Matt-Int/hexmap.el#5]
Given a specification of `q1,r1->q2,r2`, draw a path from the centre of the hex specified in the first Q R axial coordinate and to each subsequent hex.

**Comment by Matt-Int** [comment]
Considering instead to specify the start and end points of a road for each hex. That way we can instead treat each hex as having a feature which is a road. Instead of inputting start and ends of a road directly. This should match a more analogue approach where the map is created during play.
 
``` text
0, 0 plains green 0->1;0->2 ==>

| = = = = = = = = = = = = |
|    _ _                  |
|  /  | _\                |
| /   |   \ _ _           |
| \    \_ /     \         |
|  \ _ _ /       \ _ _    |
|        \       /     \  |
|         \ _ _ /       \ |
|               \       / |
|                \ _ _ /  |
|                /     \  |
|               /       \ |
|               \       / |
|                \ _ _ /  |
|                         |
| = = = = = = = = = = = = |
```

So we can specify intra-hex start and ends of a road. Where the hex sides are numbered `0->5` starting at the top and increments by 1 clockwise.

**Comment by Matt-Int** [comment]
Closed in #4
